starknet_py/tests/e2e/client/client_test.py

```diff
@@ -167,7 +167,7 @@ async def test_call_contract(client, contract_address):
         calldata=[],
     )
 
-    result = await client.call_contract(call, block_hash="latest")
+    result = await client.call_contract(call, block_number="latest")
```

Comment:
Why was this changed?

Reply:
To avoid confusion, as `latest` / `pending` is not a valid `block_hash`.

Reply:
It's still valid in terms of `Client` interface, but I'm fine with the change.